dev-util/shc/shc-3.9.6.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A generic shell script compiler"
+DESCRIPTION="Shc takes a script, which is specified on the command line and \
+produces C source code. The generated source code is then compiled and linked \
+to produce a stripped binary executable. \
+The compiled binary will still be dependent on the shell specified in the \
+first line of the shell code (i.e shebang) (i.e. #!/bin/sh), thus shc does not \
+create completely independent binaries."
+HOMEPAGE="https://github.com/neurobin/shc"
+COPYRIGHT="2015-2017 Francisco Garcia
+	Md Jahidul Hamid"
```

Comment:
According to https://github.com/neurobin/shc/commits?author=neurobin you can add "2015-2017":
```diff
 COPYRIGHT="2015-2017 Francisco Garcia
-	Md Jahidul Hamid"
+	2015-2017 Md Jahidul Hamid"
```
I don't know, however, if the years are correct for Francisco Garcia. (But I guess this can be fixed later, if someone finds out it is not correct.)

Reply:
According to the link on gentoo Francisco is the (founder?) of shc? 
http://gpo.zugaina.org/dev-util/shc